### PR TITLE
feat: switch between current and last chat

### PR DESCRIPTION
**Pull request by Himanshu-Singh-Chauhan**
`CMD + 0`
`CTRL+0`
to open last chat session.
and move focus to chat input box, after loading.

### CODE REVIEW

**Comment by nang-dev** [comment]
testing

**Comment by nang-dev** [comment]
Beautifully executed

File: gui/src/redux/slices/stateSlice.ts [hunk 1]
```diff
@@ -458,9 +456,9 @@ export const stateSlice = createSlice({
         defaultModelTitle: payload.title,
       };
     },
-    setMostRecentChat(state, action: PayloadAction<string>) {
```
Comment (nang-dev) [inline gui/src/redux/slices/stateSlice.ts:461]: Intentional?

File: gui/src/redux/slices/stateSlice.ts [hunk 2]
```diff
@@ -458,9 +456,9 @@ export const stateSlice = createSlice({
...
-      state.mostRecentChat = action.payload;
-    },
+    // setMostRecentChat(state, action: PayloadAction<string>) {
```
Comment (nang-dev) [inline gui/src/redux/slices/stateSlice.ts:459]: why commented? 
maybe should remove or leave a comment on why its commented

Comment (Himanshu-Singh-Chauhan) [reply gui/src/redux/slices/stateSlice.ts:459]: oh yeah, I was supposed to remove it, my bad, 
that code came from Akash's draft pr, we don't need that now. 
pushed a commit to remove it.

**Review by nang-dev — APPROVED** [review]

**Pull request merged**